rio_tiler/io/base.py

```diff
@@ -917,6 +992,44 @@ def _reader(band: str, *args, **kwargs) -> Metadata:
         }
         return Metadata(**meta)
 
+    def statistics(
+        self,
+        bands: Union[Sequence[str], str] = None,
+        indexes: int = 1,
```

Comment:
make it clear that only the first internal band would be used. but we still allow indexes (e.g if you have band files with multiple internal band you could select the one you want) (this was already possible but we make it clearer by adding it in the function definition